falcon/media/base.py

```diff
@@ -7,7 +7,7 @@
 class BaseHandler(object):
     """Abstract Base Class for an internet media type handler"""
 
-    @abc.abstractmethod
+    @abc.abstractmethod  # pragma: no cover
```

Comment:
Coverage was giving me grief on this line and the one below for no good reason that I could tell.